subsys/caf/modules/Kconfig.ble_adv

```diff
@@ -35,6 +35,73 @@ config CAF_BLE_ADV_DIRECT_ADV
 config CAF_BLE_ADV_FAST_ADV
 	bool "Enable period of fast advertising"
 
+config CAF_BLE_ADV_CUSTOM_INT
```

Comment:
In commit messages we use `caf: ble_adv:` format to denote module. I think it would be nice to keep commit messages consistent.